specification/threatmodel.adoc

```diff
@@ -1,187 +1,300 @@
+
+| Secure Timetamp | Trusted timestamps | Required | Confidential supervisor
+domain qualifier for CSR accesses | Ensure TEE have consistent timestamp view |
+CoVE ABI
+
+| Debug & Profile | Trusted performance monitoring unit data | Required |
+Confidential supervisor domain context switch of perf. mon. counters  |
+Ensure TEEs get correct PMU info; prevent data leakage due to PMU information
+(fingerprint attacks) | CoVE ABI
+
+| Debug & Profile | Secure External Debug support | Required | Confidential
+supervisor domain qualifier for External debug controls | Support debug trigger
+registers for TVM | Secure Debug, CoVE ABI
+
+| Debug & Profile | Authenticated debug (Production device) | Required |
+Authorize debug via TEE RoT | Ensure hardware debug prob (e.g., JTAG SWD) is
+disabled in production | Secure Debug, CoVE ABI
+
+| Availability | TVM DoS Protection | Required | VMM retains ability to
+interrupt TVM | Prevent TVM from refusing to exit | CoVE ABI, Smsdia
+
+| Availability | VMM DoS Protection | Implementation-specific | Not in scope
+for CoVE | Prevent non-TCB components from refusing to run TEE | Not applicable
+
+| Side Channel | Protected address mapping (controlled side channel) | Required
+| Confidential mode qualifier, cryptography, MMU/MPU, MTT | Similar to memory
+remapping attacks | CoVE ABI, Security Model
+
+| Side Channel | Micro-architectural side channels (branch prediction | Required
+| uArch state flushing, entropy defenses | Prevent attacks such as meltdown or
+spectre (it is difficult to defend against such attacks in advance) | Security
+Model
+
+| Side Channel | Control channels, single-step/zero-step attacks | Required |
+uArch state flushing, entropy defenses | Prevent interrupt/exception injection
+(combined with cache side channel to leak sensitive data) | Security Model
+
+| Side Channel | Architectural cache side channel | Implementation-specific  |
+uArch state flushing, entropy defenses | Prevent shared resource contention,
+e.g. attacks such as prime probe | Security Model
+
+| Side Channel | Architectural timing side channel | Implementation-specific  |
+data independent execution latency (DIEL) operations, uArch state flushing |
+Leveraging data dependency timing channels | Security Model
+
+| Secure and measured boot | Establishes root of trust in support of attestation
+| Required | RoT unique trust chain for TEE TCB | Enforcing initial firmware
+authorization and versioning | Security Model
+
+| Attestation | Remote attestation | Required | HW RoT based PKI (trust
+assertions) via Internet | Prevent fake hardware and software TCB; Prevent
+non-TCB hardware debugging in production. | CoVE ABI, Security Model
+
+| Attestation | Mutual attestation | Implementation-specific | S/U mode  |
+Attestation to another TEE on the same platform | CoVE ABI
+
+| Attestation | Remote mutual attestation | Required | Internet |  Attestation
+to a relying party on a different platform | CoVE ABI
+
```

Comment:
how different than remote attestation?

Reply:
requires additional provisioning of the TEEs to act as relying parties. agree its a nuance - will add a clarification for now - if its confusing, can remove it.